### PR TITLE
test: fix test failures related to undo/redo

### PR DESCRIPTION
**Pull request by seve**
#715

Some tests were failing on both GHA and locally. Some were due to improper accession of test fixtures, some were due to some bp3 + playwright integration issues, and some were due to fatigue!

### CODE REVIEW

File: client/__tests__/e2e/cellxgeneActions.ts [hunk 1]
```diff
@@ -59,11 +59,11 @@ export async function scroll({
 }
 
 export async function keyboardUndo(page: Page): Promise<void> {
-  await page.getByTestId("layout-graph").press("Meta+Z");
+  await page.keyboard.press("Meta+Z");
```
Comment (seve) [inline client/__tests__/e2e/cellxgeneActions.ts:62]: we don't need a locator for these

File: client/__tests__/e2e/e2e.test.ts [hunk 2]
```diff
@@ -152,31 +152,31 @@ describe("metadata loads", () => {
...
+      expect(element).toMatchSnapshot();
 
-        await page.getByTestId(`${label}:category-expand`).click();
+      await page.getByTestId(`${label}:category-expand`).click();
 
-        const categories = await getAllCategoriesAndCounts(label, page);
+      const categories = await getAllCategoriesAndCounts(label, page);
 
-        expect(Object.keys(categories)).toMatchObject(
-          Object.keys(dataTruncate.categorical.truncate[label])
```
Comment (seve) [inline client/__tests__/e2e/e2e.test.ts:171]: Erroneous accession

File: client/__tests__/e2e/e2e.test.ts [hunk 3]
```diff
@@ -449,7 +449,7 @@ describe("graph overlay", () => {
 });
 
 test("zoom limit is 12x", async ({ page }) => {
-  goToPage(page);
+  await goToPage(page);
```
Comment (seve) [inline client/__tests__/e2e/e2e.test.ts:452]: the fact I wasn't able to find this bug shows how test fatigued I was at the end of this 😆

File: client/__tests__/e2e/e2e.test.ts [hunk 4]
```diff
@@ -678,29 +677,33 @@ for (const option of options) {
       await assertGenesetExists(genesetName, page);
       await keyboardUndo(page);
       await assertGenesetDoesNotExist(genesetName, page);
+      // if we redo too quickly after undo, the shortcut handler capture the action
+      await page.waitForTimeout(500);
```
Comment (seve) [inline client/__tests__/e2e/e2e.test.ts:681]: ugh bp3

Comment (seve) [reply client/__tests__/e2e/e2e.test.ts:681]: I thought about including the timeout in the keyboardRedo helper, but didn't want to couple the two. Might create a `undoThenAssertThenRedoThenAssert` . Thoughts?

File: client/playwright.config.ts [hunk 5]
```diff
@@ -13,7 +13,7 @@ const CZI_CHECKER = " czi-checker";
 /**
  * Set this environment variable to enable retry
  */
-const SHOULD_RETRY = process.env.RETRY !== "false";
+const SHOULD_RETRY = process.env.RETRY?.toLowerCase() !== "false";
```
Comment (seve) [inline client/playwright.config.ts:16]: RETRY=FALSE was failing

File: client/src/components/hotkeys/index.tsx [hunk 6]
```diff
@@ -31,6 +31,7 @@ const GlobalHotkeys: FC<Props> = ({ undo, redo }) => {
         global: true,
         label: "Undo.",
         onKeyDown: () => {
+          console.log("UNDO");
```
Comment (seve) [inline client/src/components/hotkeys/index.tsx:34]:
```suggestion
```

File: client/__tests__/e2e/e2e.test.ts (hunk 4 above)
Comment (tihuan) [reply client/__tests__/e2e/e2e.test.ts:681]: Creating the util function SGTM! I think we can shorten the name of the function though. Maybe just `assertUndoRedo()` would be fine?

**Review by tihuan — COMMENTED** [review]
PR looks great! Thanks so much for fixing the tests, @seve 🙏 

I guess it's failing the GHA at the moment. PTAL!

File: .github/workflows/push_tests.yml [hunk 7]
```diff
@@ -82,7 +82,7 @@ jobs:
 
   smoke-tests:
     runs-on: macos-latest
-    timeout-minutes: 20
+    timeout-minutes: 30
```
Comment (seve) [inline .github/workflows/push_tests.yml:85]:
```suggestion
    timeout-minutes: 20
```

Comment (seve) [inline .github/workflows/push_tests.yml]:
```suggestion
    timeout-minutes: 25
```

**Review by kaloster — APPROVED** [review]

**Pull request merged**